Utils/Dataflow/pyDKB/common/json_utils.py

```diff
@@ -57,6 +57,8 @@ def nestedKeys(key):
 
     String should contain keys separated by dot. If a key contains
     dot itself, the key must be put between matching quotation marks.
+    Quotation marks inside the keys (not preceding or following a dot)
+    are ignored.
```

Comment:
By "ignored" you mean "treated as ordinary symbols", right?
If I read the description without knowing how the function works (or supposed to), for a moment I might think that "ignored" means "removed from the output", like:
```
>>> nested_keys('ab.c"d.e')
['ab', 'cd', 'e']
```